modules/eco-node-health-check-operator-about.adoc

```diff
@@ -1,16 +1,16 @@
 // Module included in the following assemblies:
 //
-// * nodes/nodes/node-health-check-operator-installation.adoc
+// * nodes/nodes/eco-node-health-check-operator.adoc
 
 :_content-type: CONCEPT
 [id="about-node-health-check-operator_{context}"]
 = About the Node Health Check Operator
 
 The Node Health Check Operator deploys the `NodeHealthCheck` controller, which in turn creates the `NodeHealthCheck` custom resource (CR). The Node Health Check Operator also installs the Poison Pill Operator as a default remediation provider.
 
-The Operator uses the controller to detect the health of a node in the cluster. The controller creates a `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health. 
+The Node Health Check Operator uses the controller to detect the health of a node in the cluster. The controller creates a `NodeHealthCheck` CR, which defines a set of criteria and thresholds to determine the node's health. 
```

Comment:
Should we combine these two paragraphs somehow? They seem redundant. Something like:

`The Node Health Check Operator deploys the `NodeHealthCheck` controller to detect the health of a node in the cluster. The controller in turn creates the `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health. The Node Health Check Operator also installs the Poison Pill Operator as a default remediation provider.`